tests/integration/test_bayesian_optimization.py

```diff
@@ -182,17 +183,33 @@ def GPR_OPTIMIZER_PARAMS() -> Tuple[str, List[ParameterSet]]:
                 ),
                 id="MultipleOptimismNegativeLowerConfidenceBound",
             ),
-            pytest.param(20, TrustRegion(), id="TrustRegion"),
+            pytest.param(
+                20,
+                BatchTrustRegionBox(TREGOBox(ScaledBranin.search_space)),
+                id="TREGO",
+            ),
```

Comment:
@vpicheny @hstojic double checking; are we okay with the instantiation of the rule being a bit more complex now? So, instead of `TrustRegion()`, we now need `BatchTrustRegionBox(TREGOBox(ScaledBranin.search_space))`.

Reply:
yes I think this is fine